Rover/sailboat.cpp

```diff
@@ -272,47 +270,38 @@ void Sailboat::get_throttle_and_mainsail_out(float desired_speed, float &throttl
         // rotating sails can be used to reverse, but not in this version
         mast_rotation_out = 0.0f;
     } else {
-        // + is wind over starboard side, - is wind over port side
-        float wind_dir_apparent_signed = degrees(rover.g2.windvane.get_apparent_wind_direction_rad());
-        float wind_dir_apparent_abs = fabsf(wind_dir_apparent_signed);
-        float wind_dir_apparent_sign = is_equal(wind_dir_apparent_abs, 0.0f) ? 0.0f : wind_dir_apparent_signed / wind_dir_apparent_abs;
-
-        // compute absolute sail rotation
-        float mast_rotation_angle = 0.0f;
-        bool mast_rotation_necessary = true;
+
         if (wind_dir_apparent_abs < sail_angle_ideal) {
             // in irons, center the sail.
-            mast_rotation_angle = 0.0f;
-        } else if (wind_dir_apparent_abs < (90.0f + sail_angle_ideal)) {
-            // use sail as a lift device, at ideal angle of attack
-            // (but depower to prevent excessive heel)
-            float power_coefficient = 1.0f - pid_offset/100.0f;  // pid_offset==0.0f: no restrictions on power; ==100.0f: depower entirely
-            mast_rotation_angle = wind_dir_apparent_abs - sail_angle_ideal*power_coefficient;
+            mast_rotation_out = 0.0f;
+
         } else {
-            // use sail as drag device
-            mast_rotation_angle = 90.0f;
-            // but avoid wagging the sail as the wind oscillates
-            // between 180 and -180 degrees
-            if (wind_dir_apparent_abs > 135.0f &&
-                is_equal(fabsf(SRV_Channels::get_output_norm(SRV_Channel::k_mast_rotation)), 1.0f)) {
-                // sail is already at either 90 or -90 degrees, leave
-                // it where it is; drag is identical both ways
-                mast_rotation_necessary = false;
+
+            float mast_rotation_angle;
+            if (wind_dir_apparent_abs < (90.0f + sail_angle_ideal)) {
+                // use sail as a lift device, at ideal angle of attack, but depower to prevent excessive heel
+                mast_rotation_angle = wind_dir_apparent_abs - sail_angle_ideal * MAX(1.0f - pid_offset*0.01f,0.0f);
```

Comment:
It would be really good to explain _somewhere_ what `pid_offset` is and what is the expected range is. Without that explanation, `MAX` an `*0.01f` are confusing here.